apps/frontend/core-dapp/src/components/Menu.tsx

```diff
@@ -57,9 +80,12 @@ const Wrapper = styled.div<Props>`
       border-radius: ${({ theme }): number => theme.borderRadius}px;
       padding: 0;
     }
-    .ant-dropdown-menu-item-active {
-      background-color: ${({ theme }): string => theme.color.neutral7};
-    }
+    ${({ keepBackgroundColorOnHover }): FlattenInterpolation<ThemeProps<DefaultTheme>> => {
+      if (keepBackgroundColorOnHover) {
+        return keepBackgroundOnHover
+      }
+      return hoverBackground
+    }}
```

Comment:
Alternative shorter and cleaner code IMO

```suggestion
    ${({ keepBackgroundColorOnHover }): FlattenInterpolation<ThemeProps<DefaultTheme>> => keepBackgroundColorOnHover ? keepBackgroundOnHover : hoverBackground
    }
```